http/src/main/java/com/redhat/lightblue/client/http/LightblueHttpClient.java

```diff
@@ -272,6 +262,27 @@ public LightblueResponse data(LightblueRequest lightblueRequest) throws Lightblu
     }
 
     @Override
+    public BulkLightblueResponse bulkData(AbstractBulkLightblueRequest<AbstractLightblueDataRequest> request) throws LightblueException {
+        long t1 = new Date().getTime();
+        try {
+            String responseBody = httpTransport.executeRequest(request, configuration.getDataServiceURI());
+
+            if (LOGGER.isDebugEnabled()) {
```

Comment:
No need for LOGGER.isDebugEnabled()
For log strings, use "blahblah {}", value, instead of string addition